mcp/src/main/java/io/modelcontextprotocol/spec/McpSchema.java

```diff
@@ -44,7 +44,7 @@ public final class McpSchema {
 	private McpSchema() {
 	}
 
-	public static final String LATEST_PROTOCOL_VERSION = "2024-11-05";
+	public static final String LATEST_PROTOCOL_VERSION = "2025-03-26";
```

Comment:
The LATEST_PROTOCOL_VERSION unfortunately is used for all client and server transports and raising it to `2025-03-26` would likely cause misbehaviors. 
One temporal solution is to let the transport implementation define the protocol version something like this suggestion: https://github.com/modelcontextprotocol/java-sdk/pull/404

Reply:
This is not something I wanted to deal with in this PR but I guess we have to. I'm not sure how #404 addresses this? The protocol version handling in McpAsyncServer deals with the JSON-RPC MCP domain-level checks and that's what I think was failing when I stumbled upon the need to update this. I think we just have the "latest" concept so it is accurate. Please advise how we can go about this if you have more insight, I was unfortunately quite ignorant to this aspect :(